src/downloadmanager.cpp

```diff
@@ -927,6 +927,31 @@ void DownloadManager::visitOnNexus(int index)
   }
 }
 
+void DownloadManager::openFile(int index)
+{
+  if ((index < 0) || (index >= m_ActiveDownloads.size())) {
+    reportError(tr("VisitNexus: invalid download index %1").arg(index));

```

Comment:
change error to be "OpenFile:..."

Reply:
Fixed. (This is also an issue with openInFolder below (which this `void` is basically a modified copy/paste of), I'll make another PR to fix that. :))